test/percy/spec.js

```diff
@@ -48,7 +48,7 @@ const PropValue = styled.span`
 `;
 
 const Box = styled.div`
-  background-color: ${props => (props.inverted ? '#111' : '#eee')};
+  background-color: ${props => props.theme.colorSurface};
```

Comment:
here we can use the background colour of the app instead.

Reply:
Q: is `inverted` supposed to be the same as "dark". Or we just happen to use the dark theme as the "inverted" one?